bluetooth/src/main/java/me/aflak/bluetooth/Bluetooth.java

```diff
@@ -478,7 +513,8 @@ public void run() {
                 try {
                     socket.connect();
                     connected = true;
-                    receiveThread = new ReceiveThread(socket, device);
+                    ReceiveThread receiveThread = new ReceiveThread(socket, device);
```

Comment:
Also, we should be able to differentiate messages from callbacks. We should know which socket triggered which callback.

Reply:
I need it. Do you have update for this?

```
bluetooth.setDeviceCallback(new DeviceCallback() {
    @Override
    public void onMessage(BluetoothDevice device, byte[] message) {
    }
}
```